cmd/protoc-gen-jsonschema/main.go

```diff
@@ -11,10 +11,11 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/chrusty/protoc-gen-jsonschema/internal/converter"
 	"github.com/golang/protobuf/proto"
 	plugin "github.com/golang/protobuf/protoc-gen-go/plugin"
 	"github.com/sirupsen/logrus"
+
+	"github.com/chrusty/protoc-gen-jsonschema/internal/converter"
```

Comment:
I appreciate that it is probably just IDE pedantry, but I'd prefer to minimise the scope of the PR to functional changes. Would you be able to remove this change from the PR?